.gitignore

```diff
@@ -51,3 +51,4 @@ ipch/
 *.userosscache
 *.sln.docstates
 *.xcodeproj/**
+EndlessSky.cscope_file_list
```

Comment:
Unrelated change

Reply:
It's going to be in every push I make because I don't want to upload that file. It should be ignored.

Reply:
Just don't include the changed `.gitignore` in the commit. You should only be commiting changes you want others to see. In this case what you probably want to do is add that path to your global `.gitignore`, which is in your homedir by default.

Reply:
(in fact, you can just gitignore the gitgnore)

Reply:
@AdamKauffman the easiest way is to just ignore them locally - i.e. patch your file `.git/info/exclude` to ignore them. For more details, both GitHub itself and stack overflow (https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally) have some good resources